code/modules/power/antimatter/containment_jar.dm

```diff
@@ -24,8 +34,8 @@
 		if(2)
 			if(prob((fuel/10)-stability))
 				explosion(get_turf(src), 1, 2, 3, 5)

```

Comment:
You probably want to remove this line and the line above it, as you're moving the explosion to its own proc below.